File: Core/Classes/CurrencySettings.cs
namespace Core.Classes
{
	public class CurrencySettings
    {
	    public string Symbol { get; private set; }
	    public string Layout { get; private set; }

	    private CurrencySettings()
	    {
	        Symbol = "";
	        Layout = "";
	    }

	    public CurrencySettings(string symbol, string layout)
	    {
	        Symbol = symbol;
	        Layout = layout;
	    }

        public static CurrencySettings Default
        {
            get
            {
                return new CurrencySettings();
            }
        }
	}
}

File: Php/entities/CurrencySettings.php
namespace entities{

	class CurrencySettings{

		private $symbol;
		private $layout;

		public function __construct($symbol = null, $layout = null){
			symbol = $symbol;
			layout = $layout;
		}

		public function getSymbol(){
			return symbol;
		}

		public function setSymbol($symbol){
			symbol = $symbol;
		}

		public function getLayout(){
			return layout;
		}

		public function setLayout($layout){
			layout = $layout;
		}

	}

}